app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,19 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.hummer_level = hummer_level
```

Comment:
The `hummer_level` attribute should be stored as a protected attribute, i.e., `_hammer_level`, to align with the task requirement of encapsulating additional parameters as protected attributes.